docs/src/model_setup/clock.md

```diff
@@ -34,6 +34,7 @@ for example, pass
 ```jldoctest
 julia> using TimesDates
 
+
```

Comment:
```suggestion

```

Reply:
This illustrates the danger of doctest(fix=true)